apps/gateway/src/utils/converters.ts

```diff
@@ -332,13 +335,14 @@ export function toOpenAICompatibleStream(
           }
 
           case "tool-call": {
-            const { toolCallId, toolName, input } = part;
+            const { toolCallId, toolName, input, providerMetadata } = part;
```

Comment:
I still find the name providerMetadata very confusing. On one side you call it providerOptions, on the other providerMetadata. Isn't it botg the same? If so, why not keep a consistent name?

Reply:
Clear, tq